libnetwork/drivers/bridge/bridge_store.go

```diff
@@ -25,17 +24,13 @@ const (
 
 func (d *driver) initStore(option map[string]interface{}) error {
 	if data, ok := option[netlabel.LocalKVClient]; ok {
```

Comment:
It'd be cleaner to change drivers' `Register` function to add a dedicated arg for passing the datastore. Is this something you're planning to do in a follow-up?

Reply:
More type-safe, sure, but that'd just be changing the shape of dependency injection. I don't think it's high priority; I'd like to leave it alone for now and wait for a wider refactoring of the driver interface to make such a change.